vllm/v1/core/sched/scheduler.py

```diff
@@ -773,14 +776,6 @@ def _make_cached_request_data(
                     req.num_computed_tokens : req.num_computed_tokens + num_tokens
                 ]
                 new_token_ids.append(token_ids)
-            scheduled_in_prev_step = req_id in self.prev_step_scheduled_req_ids
```

Comment:
@WoosukKwon I see that you have moved resumed requests to the new scheduled requests. However AFAIK these don't include running requests that weren't scheduled in the last step but also weren't preempted. This is currently needed for async scheduling correctness with the existing model runner.

Reply:
Fixed!